content/Phoenix for Rails developers - Part 2 - Bruno Vezoli.md

```diff
@@ -0,0 +1,131 @@
+# Phoenix for Rails developers: a practical example - Part 2
+
+### Introduction
+
+On the first part of this blog post we talked about how Rails and Phoenix compare on the web layer, if you haven't read it go and check it out. Following up with the blog example we'll see how data and the business layer are implemented on both frameworks.
```

Comment:
Maybe you can add a link to the first article on `check it out`.

Reply:
Yeah I was going to add how do we handle links?

Reply:
Since the link to the first part doesn't exist yet do something like `[check it out](link to the first part)`